bool has_puck
geometry_msgs/Point center
int32 color